files/image_config/caclmgrd/caclmgrd

```diff
@@ -202,6 +213,40 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                                                      (docker_mgmt_ip, self.namespace_mgmt_ip))
         return allow_internal_docker_ip_cmds
 
+    def generate_fwd_snmp_traffic_from_namespace_to_host_commands(self, namespace):
+        """
+        The below SNAT and DNAT rules are added in asic namespace in multi-ASIC platforms. It helps to forward the SNMP request coming
+        in through the front panel interfaces present in namespace to the SNMP Agent running in the linux host. The external IP address are
+        NATed to the internal docker IP address for the SNMP agent running in linux host to respond.
+        """
+        fwd_snmp_traffic_from_namespace_to_host_cmds = []
+
+        if namespace:
+            # IPv4 rules
+            fwd_snmp_traffic_from_namespace_to_host_cmds.append(self.iptables_cmd_ns_prefix[namespace] + "iptables -t nat -X")
+            fwd_snmp_traffic_from_namespace_to_host_cmds.append(self.iptables_cmd_ns_prefix[namespace] + "iptables -t nat -F")
```

Comment:
caclmgrd only deals with the "filter" table of iptables. I'm not sure if the "nat" table should be modified in here, or if it is better suited for a separate application. For instance, the "mangle" table is modified in hostcfgd: https://github.com/Azure/sonic-buildimage/blob/master/files/image_config/hostcfgd/hostcfgd#L70

Reply:
Thanks @jleveque for the comments .. As we discussed, I have added this change here as caclmgrd manages the static/dynamic access control rules for a device and good to have all the rules pertaining to handling control traffic for a device in common place and controlled by a common daemon. We could add support for 'nat' table of iptables as well here ?

These NAT rules added here will be applied **only once** when the caclmgrd starts up and will not be reapplied when any control plane acl changes. Also the new rules added here are applied only to linux network namespace -- and not really applicable to host, so I am not sure of adding it in hostcfgd either.

Reply:
@jleveque in my opinion i agree these are not filter rules but having here makes it's logical as other rules regarding  host <->namespace communication are present here and traffic hitting this NAT rules is subject to those filter rules also so keeping them together makes it easier to understand and if any future update is needed we can update both nat and filter set accordingly.

Reply:
I understand your points. I won't block this PR. Maybe we can discuss a refactoring in the future.

Reply:
Thanks Joe !